### ISSUE TITLE
Post card

### ISSUE THREAD

**Issue by lionirdeadman** [beehaw-org/ui-project#5]
Used as a #12 
- [ ]  Username   
- [ ]  Community posted on   
- [ ]  Title of post   
- [ ]  Image thumbnail   
- [ ]  Score   
- [ ]  Time   

Design proposal: https://design.penpot.app/#/view/3d915f59-a8d9-81d6-8002-ddf02fbc8739?page-id=3d915f59-a8d9-81d6-8002-ddf02fbc873a&section=interactions&index=0&share-id=3d915f59-a8d9-81d6-8002-ddfa02c15a61

**Comment by creesch** [comment]
As discussed on github, the initial impression I am having of this is good. A few considerations that I feel are important before we lock this in: 

- Probably good to do a few mockups of entire listings with the correct background color as well. Dark elements on white background stand out nicely. On a darker background they might lack some contracts. 
- It would be nice to see the light versions as well. 

Additionally, also consider how these look like if the user is the OP (things like, edit and delete).

**Comment by lionirdeadman** [comment]
Yeah, tweaks will probably be needed once more of the pieces are made

I think light mode should not be included in mockups just because it doubles the work needed for any mockup. I think most can imagine them with white but when implementing, we'll probably need to review some details with the light theme.

As for more actions like user-specific actions or mod actions, I think that can all fit in the "More" button as these don't need easy access.

**Comment by orowith2os** [comment]
I like it, this looks like it could be very adaptive - easily usable on both mobile and desktop.

Having user-specific options under another menu makes sense, I think; coming from Mastodon and other social medias, it would be familiar. 

The lack of icons is something I don't like, though. That might conflict with the design here, so it's not a big worry. Just something I think could be discussed a bit more.

**Comment by lionirdeadman** [comment]
The lack of icons is both because, well, I can't draw icons for my life but also because this is just a very easy way to have great accessibility (which is also why I used Atkinson Hyperlegible for text).

**Comment by orowith2os** [comment]
Fair enough, I'll save a more icon-focused path for another UI. I do think having them available would make it easier to immediately know what something does, though. I don't think a mixed design here would be too viable?

Also: just how small *could* the UI elements get here? There's a wide variety of platforms, especially mobile; there's one example for mobile, but it can possibly get even smaller than that. Could the UI adapt to be more vertical in those cases?

**Comment by lionirdeadman** [comment]
I think it should be easy to resize as needed though once all the buttons on the button squish, that's likely the limit on width.

Also for icons, some actions are pretty hard to have an icon for - most people don't understand the crosspost icon on Lemmy-ui for example. The save and more options could probably be icons. I do wonder if that'd look more inconsistent next to "Crosspost".

**Comment by creesch** [comment]
>  I do think having them available would make it easier to immediately know what something does, though. 

I remember seeing a few articles about icon use and some studies done on them. It has been a while so I am struggling to actually find the information. But it made a good case that this isn't always the case, certainly not for icons that are not widely adopted.  

Also speaking personally. Icons, certainly an overabundance of them for me, work like distracting clutter. An extreme example is new reddit where everything seems to have an icon.  I find that I simply have trouble parsing a listing there because of this. 

Of course, it is also a matter of application. On reddit there is a lot more going on, and the icons are comically oversized compared to the text.

**Comment by kalanggam** [comment]
I've been working on some mockups for post cards on mobile. You can view them here:

https://design.penpot.app/#/view/3d915f59-a8d9-81d6-8002-de4c24a01be7?page-id=f19232a9-403b-8086-8002-de2e7bf8ca3e&section=interactions&index=0&share-id=c65f3168-697c-808c-8002-dedc2e9d5d72

**Comment by zephyrix** [comment]
I made a quick mockup https://play.tailwindcss.com/1m6ldHRMmb

**Comment by dnswrsrx** [comment]
@kalanggam would you be able to share the source of the icons for your mockups?
